src/backend/opencl/kernel/mean.hpp

```diff
@@ -278,7 +278,7 @@ void mean_dim(Param out, Param in, Param inWeight, int dim)
         groups_all[dim] = 1;
         mean_dim_launcher<Ti, Tw, To>(out, owt, tmpOut, tmpWeight, dim, threads_y, groups_all);
     } else {
-        Array<Tw> tmpWeight = createEmptyArray<Tw>(0);
+        Param tmpWeight;
```

Comment:
There are similar Array declarations on Lines 479 & 579 in the same source file. Can you please correct them as well.

Reply:
Addressed